modules/tensor_mechanics/doc/content/source/materials/ADComputeGreenLagrangeStrain.md

```diff
@@ -0,0 +1,22 @@
+# ADComputeGreenLagrangeStrain
+
+!syntax description /ADMaterials/ADComputeGreenLagrangeStrain<RESIDUAL>
+
+The Green-Lagrange strain $E$ is a non-linear total strain
+
+$$
+E=\frac12 \left(F^TF-1\right)
+$$
+
+Where $F$ is the deformation gradient tensor. It can be combined with
+ADComputeLinearElasticStress to compute a second Piola-Kirchhoff stress (use teh
```

Comment:
teh -> the

Reply:
I'll clean this up in a follow on PR. The next line has another typo.